zip-1014.rst

```diff
@@ -24,13 +24,26 @@ Terminology
 The key words "MUST", "MUST NOT", "SHALL", "SHALL NOT", "SHOULD", and "MAY"
 in this document are to be interpreted as described in RFC 2119. [#RFC2119]_
 
+The term "network upgrade" in this document is to be interpreted as
+described in ZIP 200. [#zip-0200]_
```

Comment:
Could you also change this to say `as described in ZIP 200 and the Zcash trademark agreement?` (including this link) https://www.zfnd.org/about/contracts/2019_ECC_ZFND_TM_agreement.pdf
I understand that the term "network upgrade" has inconsistent meaning when one compares ZIP 200 to the trademark agreement, and it's something the Foundation and ECC should fix. But in the mean time I think it's important to reference both.

Reply:
ACK.

Reply:
That appears to be a versioned document that is subject to change. Is this the intent? :

> The term "network upgrade" in this document is to be interpreted as described in ZIP 200 [#zip-0200]_ and the Zcash Trademark Donation and License Agreement ([#trademark]_ or successor agreement).

Reply:
Thanks @daira — do you mean that there's the addendum "or successor agreement?" Yes that is intentional, in the case that the TM agreement is superseded or updated with amendments.